src/map_plot.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from folium.features import DivIcon
+import folium
+import json
+import sys
+
+# Given a VROOM solution with latlng locations, generates a map of the paths of the vehicles (up to 15 vehicles)
+def plot_map_solution(filepath):
+    f = open(filepath, 'r+')
+    vroom_res = json.load(f)
+
+    colors = ['red', 'blue', 'green', 'purple', 'orange', 'darkred', 'lightred', 'darkblue', 'darkgreen', 'cadetblue', 'darkpurple', 'white', 'pink', 'lightblue', 'lightgreen']
+    
+    first_step = vroom_res['routes'][0]['steps'][0]
+    center_lat = first_step['location'][1]
+    center_lng = first_step['location'][0]
+
+    map_center = [center_lat, center_lng]
+    map = folium.Map(location=map_center, zoom_start=10)
```

Comment:
Depending on the problem extent, we may want a different center and zoom. For example for a country-scale problem, the solution does not fit in the initial view by default.

It would be possible to expand a bounding box while going through the routes and steps, then after that fit the map zoom to this bounding box.